integration/specs/Lookup/lookup-1.spec.js

```diff
@@ -143,4 +145,45 @@ describe('Lookup base example', () => {
         browser.keys(ARROW_UP_KEY);
         expect(option1.isVisible()).toBe(true);
     });
+    it('should set focus on input when it has an option selected and clicks the label', () => {
+        const lookup = new PageLookup(LOOKUP);
+        lookup.click();
+        lookup.setQuery('a');
+        lookup.waitUntilOpen();
+        const option1 = lookup.getOption(0);
+        option1.click();
+        expect(lookup.hasFocusValueInput()).toBe(false);
+        lookup.clickInputLabel();
+        expect(lookup.hasFocusValueInput()).toBe(true);
+    });
+    it('should set focus on input when it has an option selected and clicks the input', () => {
+        const lookup = new PageLookup(LOOKUP);
+        lookup.click();
+        lookup.setQuery('a');
+        lookup.waitUntilOpen();
+        const option1 = lookup.getOption(0);
+        option1.click();
+        expect(lookup.hasFocusValueInput()).toBe(false);
+        lookup.clickValueLabel();
+        expect(lookup.hasFocusValueInput()).toBe(true);
+    });
+    it('should set focus on clear button when it has an option selected and press tab key', () => {
```

Comment:
should set focus on remove selected option button when it has an option selected and press tab key